packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -1626,16 +1626,14 @@ describe('network stubbing', function () {
       }).then(function () {
         $.post('/post-only', 'quuz').done((responseText) => {
           expect(responseText).to.contain(body)
-
-          done()
```

Comment:
Fixes [23145](https://github.com/cypress-io/cypress/issues/23145). We didn't need `done()` here -- passing and failing test cases work as expected here without calling `done()`